src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -908,6 +917,7 @@ public PreferenceSettings()
             ReadNotificationIds = new List<string>();
             DynamoPlayerFolderGroups = new List<DynamoPlayerFolderGroup>();
             backupLocation = string.Empty;
+            defaultTimeOut = 100;
```

Comment:
perhaps you should give this a more specific name if you intend it to be in the preferences for a specific call - like notifications default timeout.